README.md

```diff
@@ -0,0 +1,17 @@
+eLife Search
+=============
+
+This project is using elasticsearch to index content of the eLife articles and provide a full-text search for Journal.
```

Comment:
It's important that we state that docker is just used to improve the developer experience and that to reliably recreate any issue experienced in ci or prod we should continue to use builder.

Reply:
Added that in my last commit